frontend/components/commons/sidebar/SidebarMenu.vue

```diff
@@ -95,12 +97,14 @@ $sidebar-button-size: 45px;
   top: 0;
   width: $sidebar-button-size;
   min-width: $sidebar-button-size;
-  min-height: 100vh;
+  min-height: calc(100vh - $topbarHeight);
   min-width: $sidebarMenuWidth;
   background: palette(grey, 700);
   box-shadow: none;
   pointer-events: all;
   transition: box-shadow 0.2s ease-in-out 0.4s;
+  display: flex;
+  flex-direction: column;
```

Comment:
Lisibility: in a css selector, it's recommend to put the "important informantion" first and the rest. For example, the display and direction or more important than the "pointer-event: all" => always put the display:flex and flex-direction on top of the selector.